README.md

```diff
@@ -49,7 +47,51 @@ et [make](https://fr.wikipedia.org/wiki/Make).
 - Une fois ce compte créé et le joueur/MJ logué, il peut aller sur `ecran.php` et avoir accès à l'ensemble des commandes.
 - Le mode de fonctionnement ensuite est détaillé lors d'une partie ici : https://www.youtube.com/watch?v=bUFo1yhHT7E
 
-## In english
 
-TODO
+## 🇺🇸 English readme
+
+### WARNING
+
+⚠️ This software contains several vulnerabilities and the passwords are not secret at all! Use at your own risk.
+
+### Purpose
+
+This program lets you create an "AT RPG" (Asynchronous Tactical Role-Playing Game) on your own server where hundreds of players can take decisions collectively by voting. It was designed to be used on Twitch.
```

Comment:
```suggestion
This program lets you create at "AT RPG" (Asynchronous Tactical Role-Playing Game) on your own server where hundreds of players can take decisions collectively by voting. It was designed to be used on Twitch.
```

Reply:
Je ne comprends pas. Peut-être :
> the "AT RPG"

Reply:
Même FibreTigre n'a pas su dire ce que voulait dire le "T", j'ai déduis que c'était tactical. Sinon on peut réfléchir à un nouveau nom en français plus parlant ?

Reply:
Je ne sais pas non plus mais je propose de garder l'article défini comme pour:

> le JDR « AT RPG »

Du coup pour moi ça faisait un `at` plutôt que `a`.

Reply:
"the" est un article défini ; "at" est une préposition.

Reply:
Vu que c'est un détail et que nous ne parlons pas vraiment de la même chose, c'est sans doute sans importance. J'approuve donc.